olp-cpp-sdk-authentication/src/SignInResultImpl.cpp

```diff
@@ -91,5 +101,70 @@ const std::string& SignInResultImpl::GetScope() const { return scope_; }
 
 bool SignInResultImpl::IsValid() const { return is_valid_; }
 
+std::string SignInResultImpl::ParseJwtToken(const std::string& token) {
+  const std::string::size_type first_dot = token.find_first_of('.');
+  if (std::string::npos == first_dot) {
```

Comment:
That's an interesting expression order :) I couldn't find anything on that in the guidelines we follow but it feels unnatural to see a binary expression ordered as `constant == variable` vs the more common `variable == constant`, e. g. `nullptr != my_ptr` vs. `my_ptr != nullptr`. The latter is more common than the former, isn't it?.